index.html

```diff
@@ -277,8 +277,9 @@ <h2>Methods</h2>
                   </li>
                   <li>
                     <p class="fingerprint">If <var>deviceId</var> is not
-                    <code>""</code> and matches an id previously exposed by
-                    {{MediaDevices/selectAudioOutput}} in an earlier browsing
+                    <code>""</code> and matches an audio output device id previously
+                    exposed by {{MediaDevices/selectAudioOutput}} or
+                    {{MediaDevices/enumerateDevices()}} in an earlier browsing
```

Comment:
If a deviceId is exposed by selectAudioOutput, it would be exposed in enumerateDevices, should the web application calls enumerateDevices, just like getUserMedia.
I would either mention solely enumerateDevices, or mention both selectAudioOutput and getUserMedia.